### ISSUE TITLE
BoostedTreesClassifier

### ISSUE THREAD

**Issue by manoj1992** [tensorflow/privacy#64]
Does TF Privacy support tree-based classification models for example: tf.estimator.BoostedTreesClassifier?
Since, as per my understanding the existing DP optimizers are all Gradient Descent based.

**Comment by npapernot** [comment]
That's correct, all of our optimizers are currently based on gradient descent. However, you may be able to use something like PATE for trees: https://arxiv.org/abs/1610.05755
